scss/util/_breakpoint.scss

```diff
@@ -147,8 +147,10 @@ $breakpoint-classes: (small medium large) !default;
   @for $i from 1 through length($values) {
     $value: nth($values, $i);
     $str: breakpoint($value);
-    $bp: index($-zf-breakpoints-keys, $value);
+    $bp: index($-zf-breakpoints-keys, nth($value, 1));
```

Comment:
👍 Yes, this is consistent with the fix I had in mind

Reply:
that's the same fix I was thinking abt ;)